.github/workflows/iojs.yml

```diff
@@ -0,0 +1,69 @@
+name: iojs-ci
```

Comment:
This makes cross plat testing so much easier for Node

I switched to `setup-node` to enable cross paltform node setup, but that doesnt' support iojs. Splitting this out here, and only testing iojs on linux makes the most sense to me.